internal/directives/argocd_revisions.go

```diff
@@ -114,12 +114,12 @@ func (a *argocdUpdater) getDesiredRevisionForSource(
 			chartName,
 		)
 		if err != nil {
+			if _, ok := err.(freight.NotFoundError); ok {
```

Comment:
```suggestion
			if errors.Is(freight.NotFoundError) {
```

(applies to all of them)